### DISCUSSION TITLE
Log Entry Detail not created using SYNCHRONOUS_DML

### DISCUSSION THREAD

**Discussion by cbcruz74** [jongpie/NebulaLogger#511] [Q&A]
Hi, we could not get the Log Entry Tag records to be created using SYNCHRONOUS_DML. Log and Log Entry are being created as expected.
The Log Entry Tag records are created with EVENT_BUS only.

Is this working as designed?

**Comment by jongpie** [comment]
Hi @cbcruz74, sorry for the late response! I was able to reproduce this issue easily (using the Apex code below), and it seems like there's a bug that's preventing the tags from being saved. I'm not 100% sure where the issue is, but I've created issue #526 to track this bug.

```java
Logger.info('hello').addTag('world');
Logger.saveLog(Logger.SaveMethod.SYNCHRONOUS_DML);
```

**Comment by jongpie** [comment]
@cbcruz74 this should now be [fixed in release `v4.11.7`](https://github.com/jongpie/NebulaLogger/releases/tag/v4.11.7) 🥳
